src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -229,7 +229,7 @@
 
     context 'without build results' do
       before do
-        Buildresult.stubs(:find_hashed).returns(nil)
+        allow(Buildresult).to receive(:find_hashed).and_return(nil)
```

Comment:
the `.and_return(nil)` is not needed, just:

`allow(Buildresult).to receive(:find_hashed)`